test/labeling/apply/test_lf_applier.py

```diff
@@ -7,9 +7,8 @@
 import pytest
 from dask import dataframe as dd
 
-from snorkel.labeling.apply import LFApplier, PandasLFApplier
+from snorkel.labeling import LFApplier, PandasLFApplier, labeling_function
 from snorkel.labeling.apply.dask import DaskLFApplier, PandasParallelLFApplier
```

Comment:
Dask stuff should be two levels deep right?

Reply:
no, it's not an essential library

Reply:
https://github.com/HazyResearch/snorkel/blob/redux/requirements.txt#L9